rmgpy/molecule/pathfinderTest.py

```diff
@@ -28,6 +28,7 @@
 ###############################################################################
 
 import unittest
+from external.wip import work_in_progress
```

Comment:
Can we clean up the git history here so we don't have unnecessarily overlapping changes?

Reply:
done.